src/mono/Directory.Build.props

```diff
@@ -1,25 +1,17 @@
 <Project>
   <PropertyGroup>
-    <SkipImportArcadeSdkFromRoot>true</SkipImportArcadeSdkFromRoot>
-  </PropertyGroup>
-  <Import Project="..\..\Directory.Build.props" />
-
-  <!-- Set default Platform -->
-  <PropertyGroup>
+    <!-- Set default Platform -->
+    <Platform>$(TargetArchitecture)</Platform>
```

Comment:
This should either stay after `<TargetArchitecture Condition="'$(TargetArchitecture)' == ''">$(HostArch)</TargetArchitecture>`; or the `<TargetArchitecture Condition="'$(TargetArchitecture)' == ''">$(HostArch)</TargetArchitecture>` is unnecessary and can be deleted.